src/Components/IncomeStatement.js

```diff
@@ -20,29 +21,49 @@ const IncomeStatement = () => {
               <h4>
                 <span>Cost of Revenue</span>
                 {' '}
-                {state.costOfRevenue}
+                <span className="Value">
+                  $
+                  { state.costOfRevenue }
+                  <BsBoxArrowInRight className="IC-icone" />
```

Comment:
[OPTIONAL] As per design specifications, I think it would be best if you added the banner on the details page that shows the category name and stats.

Design:
![image](https://user-images.githubusercontent.com/38088818/217915378-1f17224e-2438-4f22-b4d2-9af3566cb747.png)

Your design:
![image](https://user-images.githubusercontent.com/38088818/217915567-c67dcfc8-7c7a-419a-9a18-381a99264ad4.png)